docs/Learning-Environment-Design-Agents.md

```diff
@@ -647,6 +649,29 @@ Notes:
 - You cannot mask all the actions of a branch.
 - You cannot mask actions in continuous control.
 
+
+#### IActuator interface and ActuatorComponents
+The Actuator API allows users to abstract behavior out of Agents and in to
+components (similar to the ISensor API).  The `IActuator` interface and `Agent`
+class both implement the `IActionReceiver` interface to allow for backward compatibility
+with the current `Agent.OnActionReceived` and `Agent.CollectDiscreteActionMasks` APIs.
+This means you will not have to change your code until you decide to use the `IActuator` API.
+
+Like the `ISensor` interface, the `IActuator` interface is intended for advanced users.
+
+The `ActuatorComponent` abstract class is used to create the actual `IActuator` at
+runtime. It must be attached to the same `GameObject` as the `Agent`, or to a
+child `GameObject`.  Actuators and all of their data structures are initialized
+during `Agent.Initialize`.  This was done to prevent an unexpected allocations at runtime.
+
+You can find an example of an `IActuator` implementation in the `Basic` example scene.
+**NOTE**: you do not need to adjust the Space Size in the Agent's
```

Comment:
```suggestion
**NOTE**: you do not need to adjust the Actions in the Agent's
```
This is just `Actions` (with `Continuous Actions` and `Discrete Branches`) in the inspector now.